### DISCUSSION TITLE
גליון 9 תרגיל 2 דינמיקה

### DISCUSSION THREAD

**Discussion by micha-sherman** [NuclearGandhi/technion_second_brain#16] [טעויות ותיקונים]
אני מנסה להבין איך קבעת שכיוון המהירות של שני הגופים לאחר ההתנגשות יהיה eBA גם כן, אם לא היה מחובר קפיץ זה היה מובן מאליו אבל החיבור של הקפיץ קצת מערער אותי פה.
![image](https://github.com/user-attachments/assets/9b49def4-c0f9-44d3-b121-8ef9a0a0cb3a)

**Comment by NuclearGandhi** [comment]
ברגע ההתנגשות, הקפיץ רפוי, ככה שהוא לא מפעיל שום כוח, כלומר תאוצה בכיוון שלו, ברגע ההתנגשות. כמובן שרגע אחרי הוא כבר מתארך ואז הכיוון הוא כבר לא BA טהור, אבל באותו הרגע מותר לנו להסיק שהכיוון הוא פשוט BA.
